main.nf

```diff
@@ -247,6 +291,8 @@ if( params.include_proteins_from_vcf) {
 } else {
     Channel
         .fromPath( params.fasta )
+        .spread(ch_samples_for_fasta)
+        .flatMap{it -> [tuple(it[1].toInteger(),it[2],it[0])]}
```

Comment:
here it 1 is converted to integer. in the other parts not. is that right

Reply:
ok right, have to check whether it is really necessary

Reply:
ok should be annotated now